src/forms/RangeField.js

```diff
@@ -33,6 +33,13 @@ export default class RangeField extends Component {
     value: _.get(this.props, 'input.value'),
   };
 
+  constructor(...arrgs) {
```

Comment:
Is it intended to be arrgs instead of args? Haven't seen that before.

Reply:
yes.. hm i copied this constructor from somewhere else and took the typo with me. will fix.